libweasyl/libweasyl/cache.py

```diff
@@ -291,7 +291,7 @@ def save(self, value):
         # turn this into a string because yajl will try to represent this in
```

Comment:
This can be removed now, since the built-in json module doesn’t do that.

Reply:
might as well clean it up before merging...? that way if we don't move away from json, we aren't left scratching our heads as to why this is still here in a year :)

Reply:
True! We do need to remember to clear the cache when this is deployed now.